### PR TITLE
Restore Issues action API

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/issue/InternalRubyIssueService.java [hunk 1]
```diff
@@ -314,6 +316,20 @@ private static Date checkAndReturnDeadline(String deadLineParam, Result<ActionPl
     return result;
   }
 
+  public Result<Issue> executeAction(String issueKey, String actionKey) {
+    Result<Issue> result = Result.of();
+    try {
+      result.set(actionService.execute(issueKey, actionKey));
+    } catch (Exception e) {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/issue/InternalRubyIssueService.java]: CRITICAL Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

File: server/sonar-server/src/main/java/org/sonar/server/computation/issue/IssueLifecycle.java [hunk 2]
```diff
@@ -70,6 +76,7 @@ public void mergeExistingOpenIssue(DefaultIssue raw, DefaultIssue base) {
     raw.setAssignee(base.assignee());
     raw.setAuthorLogin(base.authorLogin());
     raw.setTags(base.tags());
+    raw.setAttributes(base.attributes());
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/computation/issue/IssueLifecycle.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 1 critical
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* CRITICAL 1 critical

Watch the comments in this conversation to review them.

**Comment by simonbrandhof** [comment]
LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* CRITICAL 1 critical

Watch the comments in this conversation to review them.

**Pull request closed without merging**